src/pool/srv_pool.c

```diff
@@ -2234,6 +2235,112 @@ transfer_cont_buf(struct daos_pool_cont_info *cont_buf, size_t ncont,
 	return rc;
 }
 
+/**
+ * Send CaRT RPC from mgmt svc (drpc) to pool svc to get container list.
```

Comment:
dRPC/mgmt svc not really in play anymore by the time we get here. In theory anyone could call this...

Reply:
Is there a particular change that should be made?

Reply:
I think you should just remove mention of the mgmt svc and dRPC here. Of course given it's just a comment, I wouldn't ask you to resubmit.

Reply:
piggybacked this with a final functional change on the patch